src/controllers/groupController.js

```diff
@@ -0,0 +1,99 @@
+import { save, getById, update, getAllGroups, remove, addUsersToGroup } from '../services/GroupService';
+import { v4 as uuidv4 } from 'uuid';
+import { bad } from '../util/constant';
+import logger from '../util/logger';
+
+export const saveGroup = async (request, response) => {
+    const groupId = uuidv4();
+    const groupName = request.body.name;
+    const groupPermissions = request.body.permissions;
+    logger.info(`saveGroup {id: ${groupId}, name: ${groupName}, permissions: ${groupPermissions}}`);
+    try {
+        const group = await save({ id: groupId, name: groupName, permissions: groupPermissions });
+        response.status(201).send(group);
+    } catch (err) {
+        logger.error(`saveGroup failed: {id: ${groupId}, name: ${groupName}, permissions: ${groupPermissions}}, message: ${err.message}`);
```

Comment:
May be I'm not right but as I understand from task description
```
Add express middlewarewhich will log all unhandled errors and return a standard message with HTTPcode 500(Internal Server Error).
```

we need to add middleware to log this 500 errors

Reply:
You are right. I done it.